src/components/ResultsMap.vue

```diff
@@ -12,9 +12,14 @@
 				:position="m.geometry.location"
 				:clickable="true"
 				:draggable="false"
-				@click="handleMarkerClick(m.name)"
+				@click="handleMarkerClick($event, m)"
+			/>
+			<GmapMarker v-if="this.selectedMarker"
+				:position="this.selectedMarker.geometry.location"
+				:clickable="false"
+				:draggable="false"
+				:icon="{ scaledSize: {width: 28, height: 45}, url: 'https://www.clker.com/cliparts/R/g/O/v/U/h/google-maps-marker-for-residencelamontagne-md.png' }"
```

Comment:
This `scaleSize` was necessary to make the new marker cover up the existing marker

Reply:
So, by "new marker" do you mean the green one, and by "existing marker" do you mean the red one?  Could you clarify this a little bit?

Reply:
Yep, that's what I mean! 

New marker = green marker that dynamically changes position based on `selectedMarker`

existing marker = All red markers that exist in the map based on the `searchResults` data

Reply:
I'm assuming the `scaleSize` prevents any issue with the user zooming in and inadvertently displaying the existing marker?